controllers/webhook.js

```diff
@@ -0,0 +1,302 @@
+'use strict'
+
+const bufferEq = require('buffer-equal-constant-time')
+const path = require('path')
+const config = require(path.join(__dirname, '/../config'))
```

Comment:
```suggestion
const config = require('../config')
```

nit: I don't think the path join is needed here